tests/common/libtest.sh

```diff
@@ -423,6 +427,9 @@ $post
 ${posttrans:+%posttrans}
 $posttrans
 
+${transfiletriggerin:+%transfiletriggerin --} ${transfiletriggerin_pattern}
```

Comment:
I'd make this:

```bash
${transfiletriggerin:+%transfiletriggerin -- ${transfiletriggerin_pattern}}
```

to be more explicit. (I guess that applies to `%post` as well).